sections/proof-of-yield/DailyYield.tsx

```diff
@@ -36,6 +41,10 @@ const DailyYield = ({ dailyStats }: DailyYieldProps) => {
     setStats(() => [...moreStats]);
   };
 
+  const validDate = (date: string) => {
+    return moment(date).isAfter("2023-04-26");
```

Comment:
What is this related too? Exclusion of outlier data? Perhaps we really should be doing this within the endpoint or source of truth?

Reply:
The endpoint returns from the beginning of contract launch (the 4/18) but we want it to begin after the first non-zero yield day (4/26)